dev/Repeater/APITests/ViewportTests.cs

```diff
@@ -425,8 +424,16 @@ public void ValidateSupportForScrollerConfigurationChanges()
             {
                 RunOnUIThread.Execute(() =>
                 {
-                    scrollers[1].IsChildAvailableHeightConstrained = (scrollOrientation == ScrollOrientation.Horizontal);
-                    scrollers[2].IsChildAvailableWidthConstrained = (scrollOrientation == ScrollOrientation.Vertical);
+                    if (scrollOrientation == ScrollOrientation.Horizontal)
```

Comment:
>scrollOrientation == ScrollOrientation.Horizontal [](start = 24, length = 49)

nit: could be simplified to 
 var isHorizontal = scrollOrientation == ScrollOrientation.Horizontal;
 scrollers[1].ContentOrientation = scrollOrientation == isHorizontal ? ContentOrientation.Horizontal: ContentOrientation.None;
 scrollers[2].ContentOrientation = scrollOrientation == isHorizontal ? ContentOrientation.None : ContentOrientation.Vertical;

Reply:
I actually had to make changes to this test because the old combination
   IsChildAvailableWidthConstrained = true + IsChildAvailableHeightConstrained = true
is no longer available.
I made it so that all Grid and Scroller controls are sized exactly the same way as before. It took me a lot of time to get there. If you want to reshuffle this test, that's fine, but I'd prefer that to be a separate check-in later.